### ISSUE TITLE
iPodVoiceOver.framework Support

### ISSUE THREAD

**Issue by herbievine** [cormiertyshawn895/Retroactive#340]
iPods have a downloadable feature called Voice Over. It just tells you the name of the song, but it's handy.

I tried doing it in iTunes, but I didn't succeed:

![Screenshot 2023-01-05 at 22 28 59](https://user-images.githubusercontent.com/41120535/210883582-d6ff18ed-7cd4-4c8c-87bb-fbebde849080.png)

![Screenshot 2023-01-05 at 22 29 10](https://user-images.githubusercontent.com/41120535/210883590-397e1c3f-4835-40a8-9850-aba210cd5518.png)

![Screenshot 2023-01-05 at 22 30 08](https://user-images.githubusercontent.com/41120535/210883647-f4d8ee89-aec9-42e5-ab6d-37b762e2b314.png)

I then found this old post, claiming it still possible to do:

https://discussions.apple.com/thread/6832046

I installed the framework but it still didn't work...

![Screenshot 2023-01-05 at 22 31 26](https://user-images.githubusercontent.com/41120535/210883869-61eaa166-33f5-4d59-a4fa-2effd607fcb7.png)

Is there a workaround?

**Comment by JNGLHAUS** [comment]
This works for me, copy and pasted from the main page:

If you try to enable VoiceOver for iPod shuffle, but see "iTunes was unable to install the VoiceOver Kit. An unknown error occurred (1701)":

[Directly download the VoiceOver 1.4.2 installer here](https://swdist.apple.com/content/downloads/29/37/041-91732-A_LSWLP6NLRV/bk8l36k29x8d146doiwcvuv8qemu3mzxwk/VoiceOver.pkg).
Install the downloaded VoiceOver.pkg.
Quit and re-open iTunes.

**Comment by cormiertyshawn895** [comment]
Retroactive has been discontinued, and does not support macOS Sequoia or later. You should transition from Retroactive to a wide range of supported apps, many of which are built into macOS or free to download. [Learn how to transition from Retroactive to supported apps](https://github.com/cormiertyshawn895/Retroactive/blob/master/TRANSITION.md).
